source/.pylintrc

```diff
@@ -145,6 +145,7 @@ disable=print-statement,
         invalid-name,  # Eventually want to turn this on, just have a lot of errors currently, so didn't want to clutter things up.
         bare-except,
         len-as-condition,
+        pointless-string-statement   # To allow multi-line string comments
```

Comment:
Where does this show up?

Reply:
This lets us use multiline strings as comments.

Reply:
Got that part.  I just didn't initially see where they were being used not as a function or class description.

For the record, here's an example:
https://github.com/microsoft/MLOS/pull/130/files#diff-72efa0a3dd17279337c95252efb4376bc4f6bb6efc8cc1897dacae6b4a1c87c7R202